exceptionsNlistcomp/exec&listcomp.py

```diff
@@ -0,0 +1,39 @@
+N = 40
+
+from Person import Person
+
+"""
+Exceptions:
+"""
+
+def is_error() -> None:
```

Comment:
naming a function with a leading word is, has etc. is used to annotate a bool return type.
so using this when is does not return a bool is considered to be bad naming.